src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -43,25 +44,39 @@ public class PersonCard extends UiPart<Region> {
     /**
      * Creates a {@code PersonCode} with the given {@code Person} and index to display.
      */
-    public PersonCard(Person person, int displayedIndex, MainWindow mainWindow) {
+    public PersonCard(Person person, int displayedIndex, MainWindow mainWindow, boolean isVisualsEnabled) {
         super(FXML);
         this.person = person;
         this.mainWindow = mainWindow;
+        this.isVisualsEnabled = isVisualsEnabled;
         id.setText(displayedIndex + ". ");
         name.setText(person.getName().fullName);
         phone.setText(person.getPhone().value);
         email.setText(person.getEmail().value);
         person.getTags().stream()
                 .sorted(Comparator.comparing(tag -> tag.tagName))
-                .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+                .forEach(tag -> {
+                    Label tagLabel = new Label(tag.tagName);
+
+                    if (isVisualsEnabled) {
+                        if ("highnetworth".equalsIgnoreCase(tag.tagName)) {
+                            tagLabel.getStyleClass().add("tag-high");
+                        } else if ("midnetworth".equalsIgnoreCase(tag.tagName)) {
+                            tagLabel.getStyleClass().add("tag-mid");
+                        } else if ("lownetworth".equalsIgnoreCase(tag.tagName)) {
+                            tagLabel.getStyleClass().add("tag-low");
+                        }
+                    }
+                    tags.getChildren().add(tagLabel);
+                });
```

Comment:
Understand that this code is repeated below for a similar effect in the DetailedView. Would it be better to implement this as a method and call it in both places instead? So that any changes in the future would not require us to change both places.

Reply:
Ok, let's look at this at v1.6.